src/main/java/co/cask/hydrator/salesforce/authenticator/Authenticator.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.salesforce.authenticator;
+
+import co.cask.hydrator.salesforce.SalesforceUtils;
+import com.google.gson.Gson;
+import com.sforce.ws.ConnectorConfig;
+import org.eclipse.jetty.client.HttpClient;
```

Comment:
eclipse? is this the right class?

Reply:
in most examples salesforce will use jetty as a server. Which is developed by eclipse.
That's why I used it. Is this problematic?